UPGRADE.md

```diff
@@ -1,5 +1,9 @@
 # Upgrading Steps
 
+## `v1.1.0` → `v1.1.1`
+
+> 🛑 this release allows discovering database instances worldwide inside your GCP project, and not only in the pod region.
```

Comment:
Please add a `### VALUES- new" section covering the new value option; see the v1.1.0 upgrade docs for an example.

Also, change the 🛑 emoji to an ℹ️ emoji, as the former is meant to denote breaking changes.